sdk/keyvault/keyvault-certificates/test/utils/recorderUtils.ts

```diff
@@ -1,5 +1,12 @@
 import { env } from "@azure/test-utils-recorder";
 import { retry as realRetry } from "./retry";
+import { isNode } from "@azure/core-http";
+import * as dotenv from "dotenv";
+import * as path from "path";
+
+if (isNode) {
+  dotenv.config({ path: path.resolve(__dirname, "../../../.env") });
```

Comment:
I think "path" is not available in browsers, please make sure that the browser tests are unaffected with this change.

Reply:
I have isNode just above :) tests pass.